addons/account/models/product.py

```diff
@@ -63,24 +63,28 @@ def get_product_accounts(self, fiscal_pos=None):
     @api.depends('taxes_id', 'list_price')
     def _compute_tax_string(self):
         for record in self:
-            currency = record.currency_id
-            res = record.taxes_id.compute_all(record.list_price, product=record, partner=self.env['res.partner'])
-            joined = []
-            included = res['total_included']
-            if currency.compare_amounts(included, record.list_price):
-                joined.append(_('%s Incl. Taxes', format_amount(self.env, included, currency)))
-            excluded = res['total_excluded']
-            if currency.compare_amounts(excluded, record.list_price):
-                joined.append(_('%s Excl. Taxes', format_amount(self.env, excluded, currency)))
-            if joined:
-                record.tax_string = f"(= {', '.join(joined)})"
-            else:
-                record.tax_string = " "
-
+            record.tax_string = self._construct_tax_string(record, record.list_price)
+    @api.model
+    def _construct_tax_string(record, price):
```

Comment:
```suggestion

    def _construct_tax_string(self, price):
```

This is not `api.model` at all since you are using `record`, which should be named `self` to respect python's naming convention, and preceded by an empty line again to follow python's PEP8 conventions